components/OrdersTable.tsx

```diff
@@ -0,0 +1,45 @@
+import { OrderDetailsFragment } from "@/saleor/api";
```

Comment:
![image](https://user-images.githubusercontent.com/5188636/137700304-4cfa2636-d4c2-45a2-a2f8-c1d3bffa0c87.png)

- right border needs to be removed
- prices should be formatted to the currency format. Check how did we extended shema for Money type at typePolicies.ts file (we added localizedAmount). Please add same solution for the TaxedMoney